src/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights/Arcus.Observability.Telemetry.Serilog.Sinks.ApplicationInsights.csproj

```diff
@@ -24,8 +24,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.20.0" />
-    <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="3.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.21.0" />
```

Comment:
Updated this one too, as v2.20 was made deprecated.

Reply:
To be honest, I think it is strange to deprecate a package when you have just the next minor update.
Must be something very wrong in that previous version.

Reply:
Probably because of the vulnerability in Newtonsoft.Json ?
https://github.com/microsoft/ApplicationInsights-dotnet/releases